<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="HYD-LAP-00639" name="com.nisum.poc.TestNG.EmployeeTest" tests="6" failures="0" timestamp="2020-05-13T17:47:06 IST" time="6.232" errors="1">
  <testcase name="validateGetEmployee" time="3.443" classname="com.nisum.poc.TestNG.EmployeeTest"/>
  <system-out/>
  <testcase name="validatecreateEmployee" time="0.000" classname="com.nisum.poc.TestNG.EmployeeTest">
    <error type="org.testng.TestNGException" message="
Parameter &#039;Method&#039; is required by @Test on method validatecreateEmployee but has not been marked @Optional or defined
in C:\Users\NISUM\.IdeaIC2019.3\system\temp-testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'Method' is required by @Test on method validatecreateEmployee but has not been marked @Optional or defined
in C:\Users\NISUM\.IdeaIC2019.3\system\temp-testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
    </error>
  </testcase> <!-- validatecreateEmployee -->
  <system-out/>
  <testcase name="validateupdateEmployee" time="1.786" classname="com.nisum.poc.TestNG.EmployeeTest"/>
  <system-out/>
  <testcase name="validateDeleteEmployee" time="1.003" classname="com.nisum.poc.TestNG.EmployeeTest"/>
  <system-out/>
  <testcase name="loginTest" time="0.000" classname="com.nisum.poc.TestNG.EmployeeTest"/>
  <system-out/>
  <testcase name="loginTest" time="0.000" classname="com.nisum.poc.TestNG.EmployeeTest"/>
  <system-out/>
</testsuite> <!-- com.nisum.poc.TestNG.EmployeeTest -->
